pinejinx.sh

```diff
@@ -98,41 +113,46 @@ install () {
 	update-mime-database /home/${USER}/.local/share/mime
 	#Update the application database
 	update-desktop-database /home/${USER}/.local/share/applications
-	read -p "Do you want PineJinx to setup an alias for ryujinx? [y/N]: " alias
-	if [ "$alias" = "y" ] || [ "$alias" = "Y" ]; then
-		makealias
-	else
-		:
-	fi
 	printf "Installation successful, launch Ryujinx from your app launcher.\n"
 	printf "Also don't forget to show your love on Patreon at https://www.patreon.com/ryujinx\n"
 }
-uninstall () {
+uninstall() {
 	printf "Uninstalling..."
 	rm -rf /home/${USER}/.local/share/Ryujinx
 	rm -rf /home/${USER}/.local/share/mime/packages/Ryujinx.xml
 	rm -rf /home/${USER}/.local/share/applications/Ryujinx.desktop
 	rm -rf /home/${USER}/.local/share/icons/Ryujinx.png
+	rm -rf /home/${USER}/.config/gamemode.ini
```

Comment:
I know this change is from the testing branch, but I don't think it's safe to assume that deleting the gamemode config is fine on every system. Same goes for overwriting it in line 75. Ideally there would be a way to pick a custom config just for ryujinx via env variable or parameter, but I'm not sure if that's an option

Reply:
Yeah I agree, but I'm simply hotfixing the script to make it work with github and don't really have any experience with shell. So I'll have to bring that one up to the maintainer.

Reply:
removing line 125 and 75 should be sufficient, I don't see the script touching the gamemode.ini anywhere else